docs/notes/versions.rst

```diff
@@ -0,0 +1,50 @@
+.. _related_versions:
+
+Related versions
```

Comment:
Maybe we can call it Related product versions or Related dependency versions, what do you think?

Reply:
As it applies (and will apply) both dependencies (build time and run time), and other repos compatibility, I think just "Related versions" may suit better. What do you think @EduPonz?

Reply:
I mean, this title is relevant and should be short because is the one displayed in the left section/menu